src/scss/components/_button.scss

```diff
@@ -0,0 +1,39 @@
+.btn{
+  padding: .5rem 1.5rem .5rem 4rem;
```

Comment:
For values that are not font-size, kindly use `px` and also for decimal values. Kindly put 0 first.
 Example: 0.5px

Reply:
Please apply this comment to all scss files 🙇

Reply:
hahahha okay po sorry hahaahha